bin/extract-sourcemap

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+// Extract a sourcemap as JSON from a given source file
+
+let { readFile } = require("fs");
+let regex = /sourceMappingURL=data:application\/json;base64,(\w+)/;
```

Comment:
Won’t this chop off trailing `=` signs?  base64 uses those for padding, if I recall correctly.